pkg/tiller/release_update.go

```diff
@@ -151,7 +157,6 @@ func (s *ReleaseServer) performUpdate(originalRelease, updatedRelease *release.R
 	if err := s.ReleaseModule.Update(originalRelease, updatedRelease, req, s.env); err != nil {
 		msg := fmt.Sprintf("Upgrade %q failed: %s", updatedRelease.Name, err)
 		s.Log("warning: %s", msg)
-		originalRelease.Info.Status.Code = release.Status_SUPERSEDED
```

Comment:
Shouldn't the old release still be superceeded?

Reply:
if the release failed and was never deployed, the previous release is still deployed.

Reply:
Oh right; this is the error handling case. This change is correct then.